### ISSUE TITLE
Pegasi Feathers Aren't Very Useful In Singleplayer, Here's My Ideas

### ISSUE THREAD

**Issue by EternaDash** [Sollace/Unicopia#367]
> Hi. Yeah I Updated The Issue To Make It More Readable.

Hello.
Pegasi Feathers Are Used For The Wings Of Icarus Primarily, Which Is Great In Theory For Multiplayer Since It Lets Non-Pegasi Fly... *However,* In Singleplayer, Since As A Pegasus You Have The Things Strapped To Your Back 24-7, Unless You're Doing Something Incredibly Specific, Pegasi Feathers Just Become More Expensive Arrow And Writing Book Material Which Is Unfortunate.

Pegasi Feathers Need A Buff, So Here Are My Ideas.

1: Wonderbolts Suit
-
> Ah, Yes. The Wonderbolts... I Don't Know Much About Them, But I Know They Look Cool.

My Idea Would Be Leather, Blue Dye, And Pegasi Feathers To Craft The Set, And With A Full-Set It'd Give You *More Control* Whilst Flying In Weather, Make You Use *Less Mana* While Flying, *Build Up Charge* For A Sonic Rainboom Faster. The Works.

Whatever It Is, It Should Be Something To Make The Pegasus In Question Want To Get It. However, To Make It More Balanced, I Say To Give It The Protection Of *Leather Armour,* And The Durability Of Somewhere *Below Iron?* Low Enough To Not Want You To Get Into Fights With It Equipped, But High Enough So It Doesn't Break In *Two Picoseconds.*

For Repairs, Leather Would Do Fine. And Just So You Can't Cheese The Repairs Thing, It Should Be **Incompatible** With The *Mending* Enchant. That Just Screws With Balance, And I Have A Lot To Say About The Enchant And Anvils But This Isn't About That, This Is About Pegasi And Their Feathers.

2: ~~Pegasi~~ Magic Arrows
-
> For This Edit, It Has Come To My Attention That Anything Using Pegasi Feathers Shall Also Be Using Gryphon Feathers. And To Avoid That Getting Confusing, I'm Just Renaming It To Magic Arrows.

The Idea Is Simple: If You Make Arrows With Pegasi Feathers Rather Than With Regular Feathers, The Arrows Would *Fly Faster,* Which- If I'm Correct, Would Also Cause It To Do *More Damage.*

To Balance It Out, These Arrows Should Be **Incompatible** With *Infinity*, And You Should Only Get 2 For Crafting Instead Of The 4 Vanilla Arrows Give You When Crafted.

You Could Also Get These Arrows Rarely In Chests In Stuff Like Dungeons Or Strongholds, Mansions Or More Commonly In End Cities, Since I Believe Non-Pegasi Shouldn't Completely *Miss Out* On A Different Arrow Variant But Rather Have Them More Scarcely.

Plus It'd Mean Pegasi Don't _Have_ To Spend Their Feathers On It If They're Saving For A *Wonderbolts Suit,* But Obviously The Choice *Is* There If They Already Have A Set, They Just Don't Care, Or Are Endgame And Have End Cities They Can Raid.

3: Infinite* Flight Potions
-
> Ah, The Asterisk. The Favourite Punctuation Mark Of Any Legal System.

You Brew A Potion With A *Golden Feather,* And The Product Is A 'Potion Of Infinite Flight'. Obviously It Wouldn't _Literally_ Last Forever, But Whilst The Effect Is Active The Pegasi Wouldn't *Lose Any Mana* While Flying, And Would Reduce The Mana Used When Boosting By Tapping R.

This Would Lose It's Effectiveness Later On When The Pegasi Levels Up And Has More Mana, But That's *The Point.* Early Game You Rely On The Potion For Long Distance Travel, And Late Game You Have A Lot Of Mana And The *Wonderbolts Suit* And So You No Longer Need To Rely On The Potion.

4: ~~Pegasi~~ Magic Brush
-
> I Guess This Is A Thing.

Simple One Here, Using A Pegasi Feather To Make A Brush Creates An *Upgraded Brush* That Has A Higher Likelihood Of Uncovering *Valuables* From Suspicious Sand And Gravel. Would Also Incentivise Exploration Even More If Attained.

...I Mean, It's Not The *Greatest* Idea Ever, But I've Had Worse, Like Mapping Descending And Diving Separately.

5: Pegasi Feather Firework Star Shape Effect Thing Why Is This Name So Long-
-
> Gonna Be Honest, It'd Make More Sense To Have Two Separate Firework Star Shapes For Gryphon And Pegasi Feathers.

Last One Because I'm Running Out Of Ideas, And So Another Simple One. *If You Really Want To,* You Can Use Your Pegasi Feathers To Make A *Unique* Firework Star Shape Effect Thing. I'm Not Sure What You Could Make It *Exactly,* I Only Have So Many Ideas, But There's Surely Something Good.

End
-
I Edited The Thing, Enjoy.

**Comment by PrinceDrowsy** [comment]
LOVE the concept of the pegasus arrows!!!! I love all these concepts in general, but I especially love the arrow one.

**Comment by EternaDash** [comment]
too long

**Comment by Sollace** [comment]
You can keep this one open.

**Comment by Sollace** [comment]
@EternaDash Can you please edit this for readability. There's some ideas in there but it's too hard to read for me to make out what it's proposing.

**Comment by DarianLStephens** [comment]
Is it even possible to get pegasus or gryphon feathers in singleplayer?
This mod doesn't seem to have much in the way of mobs right now, and I haven't encountered them as loot of any kind.

I'm personally considering adding them as occasional drops from vindicators with a datapack, since Mine Little Pony changes them to pegasi. I did something similar for the pillagers, making them drop rare carapace, since Mine Little Pony also changes those to changelings.

---
I also very much agree that this issue needs to be rewritten, because the readability is very bad. Mostly, it needs the capitals removed from everywhere that it's not appropriate.
I used to do the same thing back in primary school. I thought it was more refined, because I had just started learning about capitals and figured, the more the merrier! But it's _really_ not. It makes it very difficult to find breaks in the sentence, or where a sentence begins and ends, and really any punctuation gets completely lost in the sea of capitals, taking extra effort to look for it and parse the words.

Here's an example of how it looks when capitalization is corrected:

> Pegasi Feathers are used for the Wings Of Icarus primarily, which is great in theory for Multiplayer since it lets non-Pegasi fly... However, in Singleplayer, since as a Pegasus you have the things strapped to your back 24-7, unless you're doing something incredibly specific, Pegasi Feathers just become more expensive arrow and writing book material which is unfortunate.

Much more readable, and the punctuation is pretty much fine as-is.

**Comment by Sollace** [comment]
> Is it even possible to get pegasus or gryphon feathers in singleplayer? This mod doesn't seem to have much in the way of mobs right now, and I haven't encountered them as loot of any kind.
> 

Yes it is! Pegasus feathers are dropped by players whilst flying, gryphon feathers can be found in mineshaft chests. Gryphon feathers are the non-exclusive alternative to pegasus feathers and can be used as a replacement in recipes that require them.

**Comment by EternaDash** [comment]
> I also very much agree that this issue needs to be rewritten, because the readability is very bad. Mostly, it needs the capitals removed from everywhere that it's not appropriate.

I've Typed Like This For Years And I'm Not Stopping Just For You, Please Don't Complain About It Again It Gets Really Annoying When Everyone Mentions It. Thanks.

**Comment by DarianLStephens** [comment]
> > I also very much agree that this issue needs to be rewritten, because the readability is very bad. Mostly, it needs the capitals removed from everywhere that it's not appropriate.
> 
> I've Typed Like This For Years And I'm Not Stopping Just For You, Please Don't Complain About It Again It Gets Really Annoying When Everyone Mentions It. Thanks.

Sorry if I offended you, that wasn't my intention.
When I bring up the capitalization, I'm doing it from a place of experience, because I did the same thing years ago, and similarly adamantly refused to stop.
I don't believe people bringing it up are trying to annoy you, it's being done for very good reason.
No one else does it because it's unpleasant to read. Doing it does make you unique, but in a way that makes it harder to take what you write seriously, and hard to tell what you're saying because we're simply not trained to read like that. We use capitalization to help indicate where sentences begin, and important words to pay attention to.
Having them absolutely everywhere makes it difficult to figure out where one thought starts and another ends, and where to look to find key words.
I myself find that I keep stopping on each word, it feels like many different starts and stops, which is happening almost automatically in my brain. It's physically hard for me to read through what you write.
I know you're unlikely to stop, because there are hundreds of people doing the same thing which pop up randomly, but it may help to know exactly what is being done to people when you write this way.
